### ISSUE TITLE
VMDPE_1 error

### ISSUE THREAD

**Issue by Sangoku** [luc-github/ESP3D#293]
I am using an ESP8266 CH340G NodeMcu V3 Lua Wireless WIFI Module Connector ESP32 Development Board ESP12E Micro USB ESP8266 CP2102 Based L293D
Flashed it and it works wonderfully. Only thing im getting is this strange 
error. Any idea what could be wrong? When i do direct uart with pc over raspbery i dont get any errors. 

But when i connect to the data port i get this in console output from the printer:

```
18:19:17.346: ok T:23.25 /0.00 B:23.47 /0.00 @:0 B@:0
18:19:17.481: echo:Unknown command: "VMDPE_1:2:2981311:0:247|2981311_VMDPE"
18:19:17.481: ok
18:19:17.731: echo:Unknown command: "VMDPE_1:2:2981561:0:247|2981561_VMDPE"
18:19:17.731: ok
18:19:17.981: echo:Unknown command: "VMDPE_1:2:2981811:0:247|2981811_VMDPE"
18:19:17.981: ok
18:19:18.235: echo:Unknown command: "VMDPE_1:2:2982061:0:247|2982061_VMDPE"
18:19:18.236: ok
```

 Build is lates 2.0.x marlin on a MKS_L board. Also i am getting random pauses on the prints also im suspecting this is the cause.

**Comment by luc-github** [comment]
Marlin says `VMDPE_1:2:2981311:0:247|2981311_VMDPE ` is unknown command but this does not come from ESP3D 
a quick google give me : `Visual Studio Ide for Arduino - VMDPE `
are you using it ?

**Comment by Sangoku** [comment]
I have it installed on my pc. 
But i did not compille with it.
Only documentation i could find on it was here:
https://www.visualmicro.com/forums/YaBB.pl?num=1381139561

**Comment by luc-github** [comment]
I cannot help as I do not use Visual Studio but you may uninstall it, as may be some service send some data on your serial port

**Comment by Sangoku** [comment]
My best gues atm is that it somehow installed a debug option on the esp... will try a clean install.  
Any advice why my prints have. It prints fine for some time with rapetier server and somhow i end up with this random pauses of a second or so..  Here is a video of it happening. 
https://drive.google.com/file/d/1s5Q7cRF1WOadA5R52WakKDikyugSb-kd/view?fbclid=IwAR1rqmZCQYDJRoN-4xjkoLdaFhmOzufvsEYk_iqPW1g2lJUPJxSwcPjUSFw

**Comment by luc-github** [comment]
Yes this is serial communication issue between host and printer - host does not fill the printer buffer on time - got this in the past on Due - not only with repetier but also S3D : https://github.com/luc-github/Repetier-Firmware-4-Davinci/issues/38

Never found the root cause and never issue anymore as always print from SD to avoid such issue

**Comment by Sangoku** [comment]
Hmm does this sketch have the same issues? 
https://github.com/probonopd/WirelessPrinting 
I only need the wirless port comunication not the ui it self.

**Comment by luc-github** [comment]
No idea - never used

**Comment by luc-github** [comment]
And this is not related to ESP3D - when I go issue I do not have it and searching on internet several people got it : https://www.google.com/search?q=random+pause+during+printing+repetier

**Comment by Sangoku** [comment]
OK! it was the darn Visual Studio.. 
I downloaded the whole project again and rebuilt it again. 
VS installs his libraries in the project so even if you later compile it with Arduino IDE the debug still is there.  

As for the random stops... i think the buffer size is kinda buggy. 
I set it below the buffer size of my printer in  Raptier and it working great for now. I miss the swiftnes of the printer tough Q_Q it is not so hyper fast with moves anymore.... it is back to his regular USB speed again.
